src/main/java/io/percy/selenium/Percy.java

```diff
@@ -164,6 +170,72 @@ public void snapshot(String name, Map<String, Object> options) {
         postSnapshot(domSnapshot, name, driver.getCurrentUrl(), options);
     }
 
+    /**
+     * Take a snapshot and upload it to Percy.
+     *
+     * @param name      The human-readable name of the screenshot. Should be unique.
+     */
+    public void screenshot(String name) {
+        Map<String, Object> options = new HashMap<String, Object>();
+        screenshot(name, options);
+    }
+
+    /**
+     * Take a snapshot and upload it to Percy.
+     *
+     * @param name      The human-readable name of the screenshot. Should be unique.
+     * @param options   Extra options
+     */
+    public void screenshot(String name, Map<String, Object> options) {
+        if (!isPercyEnabled) { return; }
+        if (!(driver instanceof RemoteWebDriver)) { return; }
+
+        String sessionId = ((RemoteWebDriver) driver).getSessionId().toString();
+        CommandExecutor executor = ((RemoteWebDriver) driver).getCommandExecutor();
+
+        // Get HttpCommandExecutor From TracedCommandExecutor
+        if (executor instanceof TracedCommandExecutor) {
+            Class className = executor.getClass();
+            try {
+                Field field = className.getDeclaredField("delegate");
+                // make private field accessible
+                field.setAccessible(true);
+                executor = (HttpCommandExecutor)field.get(executor);
+            } catch (Exception e) {
+                log(e.toString());
+                return;
+            }
+        }
+        String remoteWebAddress = ((HttpCommandExecutor) executor).getAddressOfRemoteServer().toString();
+
+        Capabilities caps = ((RemoteWebDriver) driver).getCapabilities();
+        HashMap<String, String> capability = new HashMap<String, String>();
+        capability.put("browserName", caps.getBrowserName());
```

Comment:
lets make list of capabilities and then refer that list to pick these things so that we don't need to write lot of code.
```java
List<String> caps = ["browserName", "platform", "version", "osVersion"];
for(String ele : caps) {
 options.put(ele, capability.getCapability.get(ele)
}
```